npm/src/auth/index.ts

```diff
@@ -67,59 +72,3 @@ export const refreshOrReturnToken: (
     throw "Token is expired"
   }
 }
-
-export const authenticateAndGetIdentityId: (
```

Comment:
Satisfying ;)

Reply:
I know. I'm going to delete the cognito terraform next week, I'm really looking forward to it.